src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,251 @@
+        if (req.http.x-commerce-bypass-fastly-cache == "true") {return(deliver);}
+        ```
+
+1. In **Fastly Configuration** > **Backend Settings** click **Create**. Add a new backend with the following information:
+
+   - **Condition** - `req.url ~ "^/api/(.*)" ,`
+   - **Address** - **graph.adobe.io**
+   - **Priority** - **8**
+
+1. In **Fastly Configuration** click **Upload VCL to Fastly**. Click **Save Config**.
+
+## Configure headers in API Mesh
+
+<InlineAlert variant="info" slots="text"/>
+
+After adding VCL snippets in the [Fastly Setup](#configure-fastly-in-adobe-commerce), your Commerce GraphQL URL is now in the following format with no `api_key` appended: `https://graph.adobe.io/api/<meshId>/graphql`
+
+To distinguish between requests from users and requests from API Mesh, use the following configuration option to prevent Fastly from caching headers that come directly from API Mesh:
+
+```json
+"x-commerce-bypass-fastly-cache": "true" 
+```
+
+When bypassing the cache, you must also specify which headers should be preserved in the `responseConfig`:
+
+```json
+"responseConfig": {
+  "headers": [
+    "x-magento-cache-id",
+    "x-magento-tags",
+    "set-cookie",
+    "pragma",
+    "cache-control",
+    "expires",
+    "x-content-type-options",
+    "x-xss-protection",
+    "x-platform-server"
+  ]
+}
+```
+
+Using Fastly also requires all queries to be `GET` queries. If you do not use `GET` queries, they will not be cached. To enforce this in your mesh, add the following configuration option:
+
+`"useGETForQueries": true`
+
+<InlineAlert variant="info" slots="text"/>
+
+Use the `fastly-debug:1` request header to get more information from Fastly on each request.
+
+### Fastly example mesh
+
+The following example mesh specifies the headers to cache, enables the cache bypass, and sets all queries to pass as `GET` queries.
+
+```json
+{
+  "meshConfig": {
+    "responseConfig": {
+      "includeHTTPDetails": true
+    },
+    "sources": [
+      {
+        "name": "Core",
+        "handler": {
+          "graphql": {
+            "endpoint": "https://venia.magento.com/graphql",
+            "operationHeaders": {
+              "x-magento-cache-id": "{context.headers['x-magentocache-id']}",
+              "Store": "{context.headers['store']}" ,
+              "Authorization": "{context.headers['authorization']}",
+              "Content-Type": "application/json",
+              "x-commerce-bypass-fastly-cache": "true"
+            },
+            "useGETForQueries": true
+          }
+        },
+        "responseConfig": {
+          "headers": [
+            "x-magento-cache-id",
+            "x-cache",
+            "x-magento-tags",
+            "set-cookie",
+            "pragma",
+            "cache-control",
+            "expires",
+            "x-content-type-options",
+            "x-xss-protection",
+            "x-platform-server"
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+### Fastly Prefixing
```

Comment:
Does it makes sense to change this header? Although it may affect some of the caching behavior, the prefixing isn't specific to Fastly. Maybe something like the following?
```suggestion
### Cache Header Prefixing
```
I don't feel particularly strong about this so open to other ideas.